packages/faucet/bin/faucet-server.ts

```diff
@@ -4,7 +4,7 @@ import fastify from "fastify";
 import { fastifyTRPCPlugin } from "@trpc/server/adapters/fastify";
 import { http, createClient } from "viem";
 import { privateKeyToAccount } from "viem/accounts";
-import { AppRouter, createAppRouter } from "../src/createAppRouter";
+import { AppRouter, createAppRouter } from "../src";
```

Comment:
any reason for this change?

Reply:
my ide was swearing and giving out an error. Remove the changes?

Reply:
curious to hear what the error is! but yes, my preference is not using the exports file for local imports

Reply:
my new version of GoLand reacted badly to what could be shortened, had to be fixed